2024-04-05 02:07:13,581 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Configure stats pid to 1874093
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Loading settings from /home/surajracha/.config/wandb/settings
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Loading settings from /home/surajracha/ChatLLM/DataGen/Data/Gemini/Data2/Power-Mechanism-new/Forrest Cover Type/wandb/settings
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Forrest Cover Type/cov_tester_2.py', 'program_abspath': '/home/surajracha/ChatLLM/DataGen/Data/Gemini/Data2/Power-Mechanism-new/Forrest Cover Type/cov_tester_2.py', 'program': '/home/surajracha/ChatLLM/DataGen/Data/Gemini/Data2/Power-Mechanism-new/Forrest Cover Type/cov_tester_2.py'}
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:_log_setup():526] Logging user logs to /home/surajracha/ChatLLM/DataGen/Data/Gemini/Data2/Power-Mechanism-new/Forrest Cover Type/wandb/run-20240405_020713-wm94je5r/logs/debug.log
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:_log_setup():527] Logging internal logs to /home/surajracha/ChatLLM/DataGen/Data/Gemini/Data2/Power-Mechanism-new/Forrest Cover Type/wandb/run-20240405_020713-wm94je5r/logs/debug-internal.log
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:init():566] calling init triggers
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:init():616] starting backend
2024-04-05 02:07:13,582 INFO    MainThread:1874093 [wandb_init.py:init():620] setting up manager
2024-04-05 02:07:13,583 INFO    MainThread:1874093 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-05 02:07:13,584 INFO    MainThread:1874093 [wandb_init.py:init():628] backend started and connected
2024-04-05 02:07:13,587 INFO    MainThread:1874093 [wandb_init.py:init():720] updated telemetry
2024-04-05 02:07:13,591 INFO    MainThread:1874093 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-04-05 02:07:14,593 INFO    MainThread:1874093 [wandb_run.py:_on_init():2262] communicating current version
2024-04-05 02:07:15,669 INFO    MainThread:1874093 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-05 02:07:15,669 INFO    MainThread:1874093 [wandb_init.py:init():804] starting run threads in backend
2024-04-05 02:07:18,944 INFO    MainThread:1874093 [wandb_run.py:_console_start():2241] atexit reg
2024-04-05 02:07:18,944 INFO    MainThread:1874093 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-04-05 02:07:18,944 INFO    MainThread:1874093 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-04-05 02:07:18,944 INFO    MainThread:1874093 [wandb_run.py:_redirect():2186] Redirects installed.
2024-04-05 02:07:18,945 INFO    MainThread:1874093 [wandb_init.py:init():847] run started, returning control to user process
2024-04-05 02:07:18,946 INFO    MainThread:1874093 [wandb_run.py:_config_callback():1343] config_cb None None {'data_path': 'data/covtype.csv', 'eps': 0.2, 'model_path': 'Models/cov_full_expt_512_1', 'batch_size': 4096, 'batch_size_priv': 4096, 'num_epochs': 300, 'learning_rate': 0.0003, 'wandb_project': 'covertype test', 'norm': 1, 'net_depth': 1, 'device': 'cuda'}
2024-04-05 02:27:38,720 WARNING MsgRouterThr:1874093 [router.py:message_loop():77] message_loop has been closed
